### PR TITLE
updated the readme, moved some variables to the env example file, and…

### PR DESCRIPTION
**Pull request by WesRisenmay**
… made the config file more clear about where values are coming from

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,11 +1,15 @@
-UcpClientId=""
-UcpClientSecret=""
-UcpEncryptionKey=""
+LogLevel="debug"
+Port="8088"
+Env="pre" #mocked
```
Comment (emilysallstrom) [inline .env.example:3]: Can we add the possible values that are allowed here, and expand on what "mocked" means? (I say this not knowing what mocked means, just knowing we always have used the pre environment)

File: .env.example [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
-SophtronApiUserId=""
-SophtronApiUserSecret=""
+UcpClientId="" #client id from the ucw app
```
Comment (emilysallstrom) [inline .env.example:5]: Can we add the URL for obtaining these ids/keys, as well as how each key corresponds with the labels on that page? It's a little confusing that ClientId corresponds with `user_id`, and ClientSecret is `access_key` (I think)

File: .env.example [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 
-MxClientId=""
-MxApiSecret=""
+SophtronApiUserId="" # sophtron.com -> api settings -> direct auth -> UserId
+SophtronApiUserSecret="" #sophtron.com -> api settings -> direct auth -> AccessKey
+
+MxClientId="" #Client ID from the MX dashboard
+MxApiSecret="" #API Key from the MX dashboard
 
 # Add other credentials as needed...
```
Comment (emilysallstrom) [inline .env.example:15]: Can we remove this and add stubs for the other credentials? I think there's MX Prod, Finicity, and Akoya

File: README.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ cp ./.env.example ./.env
 ```
 Next, set up the values in the `.env` file:
 1. Acquire credentials for one of the supported aggregators (currently MX or Sophtron). To sign up with MX, go [here](https://dashboard.mx.com/sign_up).
-2. Fill in the relevant credentials in `./.env` file by following config options in [example/application/config.js](../example/application/config.js) (e.g., if you want to see `mx bank` working in the widget, `MxClientId` and `MxApiSecret` must be provided).
+2. Fill in the aggregator credentials in `./.env` file (e.g., if you want to see `mx bank` working in the widget, `MxClientId` and `MxApiSecret` must be provided).
```
Comment (emilysallstrom) [inline README.md:32]: Much easier to understand 👍

**Review by emilysallstrom — CHANGES_REQUESTED** [review]

File: .env.example (hunk 3 above)
Comment (WesRisenmay) [reply .env.example:15]: I'm hesitant to add those stubs without knowing where to obtain the values for them

File: .env.example (hunk 1 above)
Comment (WesRisenmay) [reply .env.example]: Judging by the response from Jishun it doesn't seem like we know all the possible values that can be use. The only information that seems relevant about what the environment does is the mocked environment. I added a comment for that

File: .env.example (hunk 2 above)
Comment (WesRisenmay) [reply .env.example:5]: Added

**Review by emilysallstrom — APPROVED** [review]

**Pull request merged**